### ISSUE TITLE
[management] Cover uninstall URL, version and extension URL

### ISSUE THREAD

**Issue by bausano** [exteranto/framework#2]

**Comment by bausano** [comment]
Actually, we already have this param in `Versioning`:
```
@Param('app.version')
```
How does this work, @kouks ?

**Comment by bausano** [comment]
Safari doesn't have an option to set an uninstall url. We should have some policy how to deal with missing browser apis. For whole module, it's easier - you can throw a warning in service provider. However with method it's tricky as if we throw a warning, it's gonna be displayed only if the method is called.

Maybe for now, I would do the same as `Cookies`, even though it's only one method:

```
public boot () : void {
  if (this.container.resolveParam('browser') === Browser.SAFARI) {
      console.warn(new NotImplementedException('@exteranto/management', 'Runtime', ['setUninstallLink']))
    }
}
```

This way developer will know about this on boot and it's going to be easier for us to keep track as well if we come up with some better solution in future.

**Comment by bausano** [comment]
closed by #23
